HW_1/TASK6.py

```diff
@@ -0,0 +1,19 @@
+# Написать программу, которая определяет, является ли билет счастливым
+sum = 0
+sum_2 = 0
+index = 0
+ticket = input("Введите шестизначный номер билета: ")
+if len(ticket) == 6:
+    for i in ticket:
```

Comment:
Обратите внимание на функцию enumerate. С ее помощью здесь можно отказаться от объявления и инкрементирования переменной `index`.
```suggestion
    for index, i in enumerate(ticket):  # хотя переменную i я бы поменял на что-то другое.
```